### DISCUSSION TITLE
[Package] Fair AI (Ethical Data)

### DISCUSSION THREAD

**Discussion by kyleaton** [great-expectations/great_expectations#4368] [Expectations]
# What Is Ethical Data / Ethical AI and Why Is It Important?

The buzz around ethical AI has been growing louder and louder. Companies are collecting and using [more data than ever before](https://www.statista.com/statistics/871513/worldwide-data-created/). And the influence of that data is growing just as fast—with
data powering decisions and actions across every industry.

It's critical that the algorithms fueling these decisions and actions are ethical ones. That's where ethical AI comes in. Ethical AI is artificial intelligence that adheres to strict guidelines (or codes of conduct) regarding fundamental values to ensure automated systems respond to situations in an ethical way. For example, an [algorithm that determines credit limits shouldn't discriminate against women](https://www.washingtonpost.com/business/2019/11/11/apple-card-algorithm-sparks-gender-bias-allegations-against-goldman-sachs/) by granting them lower limits.

## How Can Great Expectations Contribute to Ethical AI

Great Expectations facilitates ethical AI by making it faster and easier to turn codes of conduct into actual code that can be quickly applied to data. For example, one common source of bad model performance is degradation in accuracy over time. For example, an Expectation like `expect_column_label_and_predicted_pair_accuracy_to_be_between` could be used to guard against model degradation.

**Comment by austiezr** [comment]
- [Expect Table Linear Feature Importances To Be](https://github.com/great-expectations/great_expectations/blob/develop/contrib/experimental/great_expectations_experimental/expectations/expect_table_linear_feature_importances_to_be.py)

**Comment by luismdiaz01** [comment]
I am actually working on table level expectation for this topic and was wondering if there was more documentation on Table level Expectations. For example, I am trying to make sure a Y_true, and Y_pred column are passed into the expectation
